### DISCUSSION TITLE
racemates

### DISCUSSION THREAD

**Discussion by Demydoff** [abrechts/Phoenix-ELN#22] [General]
![image](https://github.com/user-attachments/assets/4f3254bd-b526-4da2-aa26-65e5298b96b8)
How can I draw racemates in the editor? Sometimes it is essential for experiments. (trans and cis racemate)

**Comment by abrechts** [comment]
While racemates containing _one_ chiral center can easily be drawn using a wavy line, the situation for racemates containing multiple chiral centers, such as yours, indeed is different. There's currently no way to draw this in ChemBytes Draw. Your third example seems to be a way of doing this with the '&1' labels, but it might not be obvious to many users (including myself) that this means a racemate. A possible workaround could be to specifically name the racemate in the protocol, by adding the "**rac**-" prefix before the reactant name in its protocol entry, e.g. "**rac-**(1R,2R)-2-aminocyclohean-1-ol".

It would be interesting, though, to fill this gap in the reaction sketch. Do you have some examples on how this is done in other chemical drawing tools? Or any other ideas?

**Comment by Demydoff** [comment]
I know that a group of chiral centers is designated as "&1" or and1, where the number is the number of the group of atoms (screenshot is Chemdrav). In articles, sometimes two structures are drawn, or, as you said, one of the structures is labeled as a racemate. You can find more detailed information here https://www.acdlabs.com/blog/enhanced-stereochemistry-labels-advanced-techniques-for-indicating-stereoconfigurations/

I understand that this is a very specific thing

**Reply by abrechts** [reply]
Thanks for your link. It was intriguing to learn about the MDL V3000 nomenclature for describing virtually all conceivable types of mixtures, I wasn't aware of it. However, since probably not everyone will be able to interpret this nomenclature, even more than 25 years after its introduction, I guess we might need to find a more pragmatic way to describe a multicenter racemate, which is probably the most common case of a more complex mixture.

Without having evaluated the possibility to implement this in ChemBytes Draw without breaking things, a pragmatic solution for this could be rectangle brackets with with a "rac" suffix around a drawn enantiomer, as in the example below. This would give immediate visual feedback also to non-experts. Do you think that this could be an option to be assed further?

![racemate](https://github.com/user-attachments/assets/996841ae-7c7d-47f1-90a2-b88cafb051fb)

**Comment by Demydoff** [comment]
This is a logical type of sketching. This is how racemates are often labeled in articles. It would be great to see this functionality in future versions of your app. Thanks for the reply!

**Reply by abrechts** [reply]
I'll let you know whenever there's some news about this topic, it's currently medium priority for evaluation.

**Reply by abrechts** [reply]
This feature is implemented now as part of the newly released [Phoenix ELN version 2.5.0](https://github.com/abrechts/Phoenix-ELN/releases/tag/v.2.5.0).

**Reply by Demydoff** [reply]
> This feature is implemented now as part of the newly released [Phoenix ELN version 2.5.0](https://github.com/abrechts/Phoenix-ELN/releases/tag/v.2.5.0).

For me, this is the best app update! Thank you!
